validator/sawtooth_validator/journal/consensus/poet/poet_consensus/poet_transaction_block.py

```diff
@@ -237,6 +238,7 @@ def is_valid(self, journal):
 
             try:
                 self.wait_certificate.check_valid(
+                    poet_enclave_module=None,
```

Comment:
This code is not used any more and it is only left in the repo until I have the new 0-8 version on feature parity.  I can mark the file somehow to indicate deprecated.